Documentation/Testing/ExtensionTesting.rst

```diff
@@ -218,23 +208,13 @@ to add root :file:`composer.json` details, turning the extension into a project
           "Lolli\\Enetcache\\Tests\\": "Tests"
         }
       },
-      "replace": {
-        "enetcache": "self.version",
-        "typo3-ter/enetcache": "self.version"
-      },
-      "scripts": {
-        "post-autoload-dump": [
-          "TYPO3\\TestingFramework\\Composer\\ExtensionTestEnvironment::prepare"
-        ]
-      },
       "extra": {
         "branch-alias": {
           "dev-master": "2.x-dev"
         },
         "typo3/cms": {
           "cms-package-dir": "{$vendor-dir}/typo3/cms",
```

Comment:
Is this not superflous here like above ? TBH don't know that argument anyway ;)

Reply:
done